swift-sdk/Internal/InAppManager.swift

```diff
@@ -186,6 +144,20 @@ class InAppManager: NSObject, IterableInternalInAppManagerProtocol {
         messagesMap[id]
     }
     
+    func isAutoDisplayPaused() -> Bool {
+        autoDisplayPaused
+    }
+    
+    func setAutoDisplayPaused(_ paused: Bool) {
+        autoDisplayPaused = paused
+        
+        if !autoDisplayPaused {
+            _ = scheduleSync()
+        }
+    }
+    
+    // MARK: - InAppDisplayChecker
```

Comment:
Not needed for +1. If you want, this one you can move outside using
`extension InAppManager: InAppDisplayChecker`